Gemfile.base

```diff
@@ -108,7 +108,7 @@ gem 'prawn-format', '0.2.1'
 gem 'prawn-layout', '0.2.1'
 gem 'swagger-ui_rails2', git: 'https://github.com/3scale/swagger-ui_rails.git', branch: 'dev-2.1.3'
 gem 'swagger-ui_rails', git: 'https://github.com/3scale/swagger-ui_rails.git', branch: 'dev'
-gem 'json-schema'
+gem 'json-schema', git: 'https://github.com/3scale/json-schema.git'
```

Comment:
ooops, another fork!

Reply:
We need it to support JSON schema draft-07.